pkg/model/config/config_test.go

```diff
@@ -295,4 +296,17 @@ var _ = Describe("Config", func() {
 		})
 
 	})
+
+	Context("HasWebhook", func() {
+		It("should return true when config has a webhook with the GVK", func() {
+			c.UpdateResources(gvk3)
+			Expect(c.Resources).To(Equal([]ResourceData{gvk3}))
+			Expect(c.HasWebhook(gvk3)).To(BeTrue())
+		})
+		It("should return false when config NOT has a webhook with the GVK", func() {
```

Comment:
Any of the following options:
```suggestion
		It("should return false when config does not have a webhook for the GVK", func() {
```
```suggestion
		It("should return false when config has no webhook for the GVK", func() {
```